tests/CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ add_dependencies(build_tests aktualizr)
 
 
 if(BUILD_SOTA_TOOLS)
-    add_dependencies(build_tests garage-push garage-deploy-deb)
```

Comment:
This is not mistake?

Reply:
The target doesn't exist anymore. @OYTIS has redone some of the packaging. It's now all done in the root `CMakeLists.txt`.